tools/devtool

```diff
@@ -532,7 +532,7 @@ ensure_ci_artifacts() {
 # Please see `$0 help` for more information.
 #
 cmd_test() {
-
+    do_ab_test=0
```

Comment:
Could we do a different subcommand? Like maybe

```sh
cmd_abtest () {
    cmd_sh "./tools/ab_test.py $@"
}
```

Reply:
I wanted to put it into its own command originally, but I ended up copying all the surrounding code (like environment variable pass through, ci artifacts download, performance tweaks), so I merged it into the existing test command instead